apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -155,6 +162,42 @@
                     "description": "Time (milliseconds) between logging info about connected peers",
                     "type" : "integer",
                     "default": 5000
+                },
+                "peer_pool" : {
+                    "type" : "object",
+                    "additionalproperties" : false,
+                    "properties" : {
+                        "select_verified_peer_probability" : {
+                            "description" : "Probability of selecting a peer from the verified pool.",
+                            "type" : "number",
+                            "minimum" : 0.0,
+                            "maximim" : 1.0,
+                            "default" : 0.5
```

Comment:
We agreed to set it to 1.0 by default. Otherwise we won't see any improvement

Reply:
Yes, but as written in the PR comment, this PR doesn't change the defaults, they will be changed in a separate PR.